IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.Tests/TestBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using IntelliTect.TestTools.TestFramework.Tests.TestData;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace IntelliTect.TestTools.TestFramework.Tests
+{
+    public class TestBase
+    {
+        protected static void ValidateAggregateException(AggregateException result, int expectedInnerExceptions, params string[] messages)
+        {
+#pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
Because you can't assume that callers have nullable reference types enabled, I would add null checks rather than supress the exception.